ruby/ql/src/queries/security/cwe-094/CodeInjection.ql

```diff
@@ -16,20 +16,9 @@
 
 private import codeql.ruby.AST
 private import codeql.ruby.security.CodeInjectionQuery
-import CodeInjectionFlow::PathGraph
+import DataFlow::DeduplicatePathGraph<CodeInjectionFlow::PathNode, CodeInjectionFlow::PathGraph>
```

Comment:
How about not re-exporting the dedup'ed pathgraph, and instead export the boilerplate-translated `flowPath` predicate? Then we'd write
```suggestion
module CodeInjectionFlowDeDup = DataFlow::DeduplicatePathGraph<CodeInjectionFlow::PathNode, CodeInjectionFlow::PathGraph>;
import CodeInjectionFlowDeDup::PathGraph
```